src/actions.js

```diff
@@ -127,6 +127,17 @@ export function fetchLocationsStr(mm, level, parent, str, first) {
   return graphql(payload, `LOCATION_LOCATIONS_${level}`);
 }
 
+export function fetchParentLocationsStr(mm, level, parentUuids, str, first) {
```

Comment:
what does mean 'str'? Can you specify more accurate name for this argument?

Reply:
and also mm?

Reply:
I extended the shortcuts.